core/src/io/Filter_File_Handle.cpp

```diff
@@ -10,42 +10,61 @@
 #include <string>
 #include <cstring>
 #include <sstream>
+#include <iostream>
 
 using namespace Utility;
 
 namespace IO
 {
-	Filter_File_Handle::Filter_File_Handle(const std::string& filename):
+	Filter_File_Handle::Filter_File_Handle( const std::string& filename, IO::VF_FileFormat format ):
 		filename(filename), iss("")
 	{
-		dump = "";
-		line = "";
-		found = std::string::npos;
-		myfile = std::unique_ptr<std::ifstream>(new std::ifstream(filename));
-		if (!myfile->is_open()) { throw Utility::Exception::File_not_Found; }
-		else {
-			Log(Log_Level::Debug, Utility::Log_Sender::IO, std::string("  Reading from Config File ").append(filename));
-		}
+        this->ff = format;
+        this->dump = "";
+        this->line = "";
+        this->found = std::string::npos;
+        this->myfile = std::unique_ptr<std::ifstream>( new std::ifstream( filename,
+                                                       std::ios::in | std::ios::binary ) );
+        
+        // set the comment tag
+        if( this->ff == VF_FileFormat::OVF ) 

```

Comment:
This could be set in the Dataparser function which calls this Class and passed into the constructor as a string, right? That would be more general than giving this class a file format.